.gitignore

```diff
@@ -3,3 +3,5 @@ __pycache__
 /dist
 /*.egg-info
 /.mypy_cache
+/.pytest_cache
+/.vscode
```

Comment:
Can you split these additions into a different commit?

Reply:
Okay I think this is done as well. Hopefully I did that right - this is my first time contributing to any kind of software project, sorry in advance for any errors I make.